README.md

```diff
@@ -1 +1,63 @@
 # jQuery Number Input Formatter
+
+A jQuery library to dynamically display a numeric input's formatted value in a
+label.
+
+This library will take the value of an HTML input, format it, and place the
+formatted representation into a label. The purpose is to give the end user an
+opportunity to more easily identify their numeric inputs for large numbers.
+
```

Comment:
This was addressed but a little further down in the README so not considered an "outdated" diff.